.gitignore

```diff
@@ -1,2 +1,3 @@
 .mypy_cache/
 .pytest_cache/
+search.json
```

Comment:
needed anymore? Running the application uses `.docker/search.dev.json`

Reply:
It prevents an accidental commit, but the file won't end up in the Docker image anyway

Reply:
I've removed search.json with constants.py.